app/jobs/assign_doi_job.rb

```diff
@@ -13,6 +13,7 @@ def run
     doi_request = DoiRequest.find(doi_request_id)
     asset = Collection.find(doi_request.asset_id)
     minted_doi = Ezid::Identifier.mint(
+      target: "http://data.lib.vt.edu/collections/" + asset.id,
```

Comment:
Should we either use a protocol relative link, or (maybe even better) just link to "https" instead of "http"?